src/mainviews/list-jobs.tsx

```diff
@@ -246,7 +252,11 @@ function ListJobDefinitionsTable(props: ListJobDefinitionsTableProps) {
     !deletedRows.has(jobDef.job_definition_id);
 
   const emptyRowMessage = useMemo(
-    () => trans.__('There are no notebook job definitions.'),
+    () =>
+      trans.__(
+        'There are no notebook job definitions. Notebook job definitions run notebooks and other files in the background on a schedule. ' +
```

Comment:
```suggestion
        'There are no notebook job definitions. Notebook job definitions run files in the background on a schedule. ' +
```

Copy edit for brevity